src/main/docs/guide/languageSupport/kotlin/ksp.adoc

```diff
@@ -0,0 +1,5 @@
+You can build Micronaut applications with Kotlin and https://kotlinlang.org/docs/ksp-overview.html[KSP]:
```

Comment:
should probably include a build example:

```groovy
plugins {
    id("org.jetbrains.kotlin.jvm") version "1.8.0"
    id("com.google.devtools.ksp") version "1.8.0-1.0.9"
    id("org.jetbrains.kotlin.plugin.allopen") version "1.8.0"
    id("io.micronaut.application")
}

dependencies {
    ksp("io.micronaut:micronaut-inject-kotlin")
    kspTest("io.micronaut:micronaut-inject-kotlin")
    ....
}
```

Reply:
see: https://github.com/micronaut-projects/micronaut-core/pull/8786/commits/a9ff12bd0c38ad295d5679948925176b6d69a476